lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -65,19 +65,6 @@ def standalone_page_title(self, request, fragment, **kwargs):
         """
         return _('Programs')
 
-    def css_dependencies(self):
```

Comment:
Why is this being removed?

Reply:
If this is removed, should this also be? https://github.com/edx/edx-platform/pull/24624/files#diff-82bda515879c96ab0641bfcbe58631c3L1891-L1902

Reply:
@abutterworth this view used two styleshhets: bootstrap/lms-main and lms-learner-dashboard, as the second is going to removed, there is no need to this part.

Reply:
yes, it should be removed too. I will update the pr.